app/styles/component/_post-search-results-item.scss

```diff
@@ -0,0 +1,46 @@
+.post-search-results-item {
+  margin: 18px 0;
+
+  // Clearfix
+  &::after {
+    clear: both;
+    content: "";
+    display: block;
+    height: 0;
+    visibility: hidden;
+  }
+
+  &__title {
+    color: #0d7675;
+    display: inherit;
+    font-size: $wds-font-size-xs;
+    font-weight: $wds-font-weight-medium;
+    letter-spacing: .03px;
+    line-height: 1;
+    margin-bottom: 9px;
+  }
+
+  &__details {
+    align-items: center;
+    color: $wds-color-dark-gray;
+    display: flex;
+    fill: currentColor;
+    font-size: $wds-font-size-xs;
+    font-weight: $wds-font-weight-normal;
+    line-height: 1;
+
+    & > svg {
+      margin-right: 6px;
+    }
+  }
+
+  &__image {
+    border: solid 1px #bed2cf;
```

Comment:
I _think_ this is supposed to be `$wds-fandom-color-mid-light-gray` even though G is off by 1 number.

Reply:
👍 Good finiding. This component will go through Design Review once it's merged and enabled on on keikotest wiki on prod.

Reply:
ff5cd58